bi-connectors/TableauConnector/src/connectionProperties.js

```diff
@@ -0,0 +1,12 @@
+(function propertiesbuilder(attr) {
+    var props = {};
+    props["user"] = attr[connectionHelper.attributeUsername];
+    props["password"] = attr[connectionHelper.attributePassword];
+
+    if (attr[connectionHelper.attributeSSLMode] == "require") {
+        props["ssl"] = "true";
+        props["sslmode"] = "require";
+    }
```

Comment:
I think `v-trustSelfSigned` and `v-region` should be included here as well, ref:
https://github.com/Bit-Quill/sql-jdbc/blob/c0ce14f5bc867b67ca1ce6641174398ba96b91ad/bi-connectors/TableauConnector/src/connectionBuilder.js#L16-L19

Reply:
Made connector more secure by removing username and password in the url as recommended in https://tableau.github.io/connector-plugin-sdk/docs/api-reference. No need for other properties here.

Reply:
Should we raise a follow-up ticket to address this?

Reply:
No. No further action is needed. Changes are in the this PR.